### DISCUSSION TITLE
「组队」基于Sui的SBT证书

### DISCUSSION THREAD

**Discussion by RandyPen** [NonceGeek/Move-Camp#207] [Amazing Things!]
# 引言

## SBT
区块链的世界里，账户地址表示用户。将一些绑定不可交易的独特Token放置到特定账户中，可视为给该账号贴上标签。圈子里的人将其称为[SBT(Soulbound Token)](https://mp.weixin.qq.com/s/U7CYJ4IOUWk1MPkztpbKXg)

[SBT综述](https://mp.weixin.qq.com/s/fUxVx9HK8J44HOf6Y1yqKA)

## 经历与能力
特定经历可以作为能力的背书。

比如，皮卡参加了亚非拉Web3研究院的官方博客阅读共学营；皮卡搭建了亚非拉Web3研究院早期的共学实践培养模式；皮卡在Web3-dApp-Camp中研发了证书式的SBT.

这些经历都在某些维度上证明了能力，区块链圈子又对链上属于用户的信息更为关注。将这些经历制作成SBT证书，赋予到用户的账户里，就成为分布在链上的简历（而不是存放在Linkedin上的简历。

# 产品设计

## 通用

- 不可交易

- 发证机构账户可以授予或撤回，但撤回操作需要支付更高昂的Gas成本，作为审查不力的代价

## 必备信息

- 发证机构的账户地址，账户地址对应的机构身份可以由其他DID服务证明

- 接收SBT用户的账户地址，身份证明同样可以采用其他DID服务

- 经历/培训活动的文字描述，培训活动的链接

- 接收SBT用户在培训中产出作品的链接，中心化或去中心化发布渠道的作品链接都可以，会生成一串哈希码作为作品验证。比如产业研究员在培训中写出的研报文章链接，程序员在代码项目中提交的git版本记录

## 交互操作

- 发证机构授予或撤销SBT证书的前端操作，包含上述必备信息的输入，能连接钱包账户地址

- 根据账户地址查看拥有SBT证书的前端页面

## 锦上添花

- 根据必备信息生成哈希码，映射到特定数字，制作生成艺术添加到SBT中

- 由持有SBT证书用户账户定制的简历页面

- 发证机构发证信息查询页面，活动、发送群体

[皮卡的原文](https://mp.weixin.qq.com/s/fmKejxolo7n7lDIxDfSVNA)

**Comment by leeduckgo** [comment]
招 1-2 名前端.

**Reply by 0xKT** [reply]
🙋🏻‍♀️

**Comment by xiegudong45** [comment]
现在还可以加入吗？

**Reply by xiegudong45** [reply]
本职工作是golang开发，目前我可以写简单的react及sui move
